### PR TITLE
Fix handling of non-dict configs

### PR DESCRIPTION
**Pull request by palango**
As seen in `AttributeError: 'int' object has no attribute 'get'`

### CODE REVIEW

File: scenario_player/tasks/base.py [hunk 1]
```diff
@@ -110,9 +110,12 @@ def __call__(self, *args, **kwargs):
         self._runner.running_task_count += 1
         self._start_time = time.monotonic()
         try:
+            timeout_s = None
+            # Config can be something else than a dictionary
```
Comment (hackaugusto) [inline scenario_player/tasks/base.py:114]: when would the config be something else? this seems a problem that should be handled in the caller site, and not here.

Comment (palango) [reply scenario_player/tasks/base.py:114]: For example in the `WaitTask`, the config is just an `int`

Comment (hackaugusto) [reply scenario_player/tasks/base.py:114]: The point is that it should be fixed there then.

Comment (palango) [reply scenario_player/tasks/base.py:114]: I guess we want to support this:
```
 - wait: 30
```
instead of 
```
- wait: {time: 30}
```

Comment (palango) [reply scenario_player/tasks/base.py:114]: I propopose to fix that later: https://github.com/raiden-network/scenario-player/issues/584

**Pull request merged**